packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.security_extension.test.ts

```diff
@@ -423,6 +424,28 @@ describe('SavedObjectsRepository Security Extension', () => {
         })
       );
     });
+
+    test(`adds created_by to the saved object when the current user is available`, async () => {
+      const userId = 'userId';
+      mockSecurityExt.getCurrentUser.mockImplementationOnce(
+        () =>
+          ({
+            profile_uid: userId,
+          } as AuthenticatedUser)
```

Comment:
__nit:__ not for this PR, but it'd probably make sense to have a `mockAuthenticatedUser` function exposed via `core-security-{common|server|browser}-mocks` or something along these lines @pgayvallet

Reply:
I updated in this pr 👍